### DISCUSSION TITLE
Where is the wal2json plug-in installed?

### DISCUSSION THREAD

**Discussion by cap-itadmin** [eulerto/wal2json#195] [Q&A] [answered]
Hello,

I would like to use the wal2json plug-in for pipelinewise.  I have an RDS postgres server and an EC2 Ubuntu instance which will be running the pipelinewise jobs.  

Should the plug-in be installed on the postgres server, or on the Ubuntu instance?

Thanks,
Karen

**Comment by eulerto** [comment, marked as answer]
`wal2json` is installed in the postgres server. It is installed in the same directory as the other postgres extensions.
